internal/cli/streams/processor/describe_test.go

```diff
@@ -0,0 +1,151 @@
+package processor
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/mocks"
+	"github.com/stretchr/testify/assert"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20241113004/admin"
+)
+
+func TestDescribeOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockProcessorDescriber(ctrl)
+
+	pipeline := []any{
+		map[string]any{
+			"$source": map[string]any{
+				"connectionName": "sample_stream_solar",
+			},
+		},
+		map[string]any{
+			"$match": map[string]any{
+				"group_id": 10,
+			},
+		},
+		map[string]any{
+			"$merge": map[string]any{
+				"into": map[string]any{
+					"db":             "testdb",
+					"coll":           "testcoll",
+					"connectionName": "atlascluster",
+				},
+			},
+		},
+	}
+
+	expected := atlasv2.NewStreamsProcessorWithStats("1", "ExampleSP", pipeline, "STOPPED")
+	expected.Stats = map[string]any{
+		"dlqMessageCount":   0,
+		"inputMessageCount": 150,
+		"inputMessageSize":  500,
+	}
+
+	mockStore.
+		EXPECT().
+		StreamProcessor(gomock.Any()).
+		Return(expected, nil).
+		Times(2)
```

Comment:
I think that separate stores should be pulled into each test to more specifically assert each test's code.